exts/cesium.omniverse/mdl/cesium.mdl

```diff
@@ -203,18 +204,18 @@ float4 alpha_blend(float4 src, float4 dst) {
 
 float4 compute_base_color(
     gltf_texture_lookup_value imagery_layers_texture,
-    color debug_color,
+    float4 tile_color,
```

Comment:
Are we moving away from `fmt` for colors in general?

Reply:
Ah, no alpha-channel support in `fmt`.

Reply:
This is MDL code, so no relation to `fmt`